.env.development

```diff
@@ -33,7 +33,6 @@ ENABLE_ACCESSIBILITY_PAGE=false
 ENABLE_PROGRESS_GRAPH_SETTINGS=false
 ENABLE_TEAM_TYPE_SETTING=false
 ENABLE_NEW_EDITOR_PAGES=true
-ENABLE_NEW_COURSE_OUTLINE_PAGE = false
```

Comment:
@Cup0fCoffee Since the page is not ready yet, I set it to 'false'. Also, left `getPagePath` as is as it is generic method to fetch url based on mfeEnabled flag so it could be used for other urls (like in header links).